rules/aws_cloudtrail_rules/my_first_custom_rule.py

```diff
@@ -0,0 +1,23 @@
+from pypanther import PantherLogType, PantherRule, PantherRuleTest, PantherSeverity
```

Comment:
What if we broke the pattern of per log_type file paths for custom rules? My hesitation is that customers will forget that they need an `__init__.py` in all new files for custom rules, and `pypanther` naturally allows customers to write more rules in less files with simpler folder structures.

cc @arielkr256 for his thoughts on the subject

Reply:
Forcing the customer to create and import a new class - in addition to what they already have to do to create the custom yaml schema - for a custom log type feel burdensome.  I am a fan of just using the string `"Custom.MyLogType"` and calling that out in the examples.  In the future, when custom schemas are defined as classes, this approach of importing should still work.

Reply:
The cool part about the class is that you won't have any typos and you will get autocomplete. It seems like some of those could outweigh the burden of a custom class

Reply:
I agree with Ariel - I think asking users to create what essentially amounts to an empty class whenever they want to use a custom log type is a bit jarring. If we had class support for entire schemas they can manage with pypanther, then that's different, but right now I think we should allow users to pass schema names as strings. Those customers who would prefer autocomplete can make an enum, and we can include an example of that in our docs

Reply:
The CustomLogType class would have all custom log types. Not one class per log type. It acts the exact same as the PantherLogType, but you can't subclass an Enum in python sadly, so we have to make a new class